test/it/client-listUsers.js

```diff
@@ -15,8 +20,18 @@ describe('client.listUsers()', () => {
   let _user;
 
   before(() => {
-    const newUser = utils.userWithPassword();
-    newUser.profile.nickName = faker.random.uuid();
+    const newUser = {
+      profile: {
+        firstName: 'John',
+        lastName: 'List-Users',
+        email: 'john-list-users@example.com',
+        login: 'john-list-users@example.com'
+      },
+      credentials: {
+        password: {value: 'Abcd1234'}
+      }
+    };
+    newUser.profile.nickName = 'johny-list-users';
```

Comment:
nit: is there a reason we don't include the `nickName` in `profile` above? If it's to show that we're adding an unmapped property, let's add a comment. It looks a little weird without the explanation.